lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -823,11 +933,16 @@ HdVP2Material::_AcquireTexture(const std::string& path)
     }
 
     bool isSRGB = false;
-    MHWRender::MTexture* texture = _LoadTexture(path, isSRGB);
+    MFloatArray          uvScaleOffset;
+    MHWRender::MTexture* texture = _LoadTexture(path, isSRGB, uvScaleOffset);
 
     HdVP2TextureInfo& info = _textureMap[path];
     info._texture.reset(texture);
     info._isColorSpaceSRGB = isSRGB;
+    info._stScale[0] = uvScaleOffset[0]; // The first 2 elements are the scale, the 2nd two elements are the offset.
+    info._stScale[1] = uvScaleOffset[1];
+    info._stOffset[0] = uvScaleOffset[2];
+    info._stOffset[1] = uvScaleOffset[3];
```

Comment:
If you use GfVec2f and also change the signature of _LoadTexture to accept two GfVec2f objects, the code here can be simplified a bit.

Reply:
I think I need them in the HdVP2TextureInfo in case we hit the texture cache. Using the GfVec2f type makes sense for sure.